### PR TITLE
 Alphabetically sort MTGO Cubes so they are easier to find

### PR DESCRIPTION
**Pull request by myyk**
Implements #6200.

I have one commit as moves and one commit as edits for easier reviewing. It's too bad that it didn't show them as moves for all the `.draft` files in github, but that's out of my control.

### CODE REVIEW

**Comment by tehdiplomat** [comment]
Is this going to cause issues for people who are in between gauntlets, or have their preferences pointed to the old names?

**Comment by myyk** [comment]
@tehdiplomat, I'm unfamiliar with Gauntlet, but as far as I can tell from playing with the UI it doesn't seem related to draft or cube so I think those are totally separate and safe if in progress.

As far as drafted decks from a cube go, the decks seem unrelated to the cube they were drafted from. And there's no in progress saving of a draft, so nothing to worry about from that front.

**Review by tehdiplomat — DISMISSED** [review]

File: forge-gui/res/cube/MTGO Legacy Cube 2017-01.dck [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [metadata]
-Name=MTGO Legacy Cube January 2017
+Name=MTGO Vintage Cube 2017-01
```
Comment (myyk) [inline forge-gui/res/cube/MTGO Legacy Cube 2017-01.dck:2]: Oops, I'll fix this

**Comment by myyk** [comment]
@myyk I triple checked all changes after fixing the 1 typo I found.

**Comment by Agetian** [comment]
Is this MR ready?

**Comment by myyk** [comment]
It is

**Review by Agetian — APPROVED** [review]

**Pull request merged**